source/agora/consensus/protocol/Nominator.d

```diff
@@ -600,14 +600,16 @@ extern(D):
         else
         {
             log.info("{}(): Tx set didn't trigger new nomination", __FUNCTION__);
-            const Height expected = this.ledger.expectedHeight(this.clock.utcTime());
-            if (expected > this.ledger.height + 1) // We are one block late
+            const Duration halfBlockDuration = this.params.BlockInterval / 2;
+            // We are half way through block interval time to externalize
+            if (this.clock.utcTime() > this.ledger.getExpectedBlockTime(this.ledger.height) + halfBlockDuration)
```

Comment:
```suggestion
            if (this.clock.utcTime() > this.ledger.getExpectedBlockTime(this.ledger.height) + 
                this.params.BlockInterval / 2)
```
Can't we just use this operation without introducing the variable?

Reply:
The `const` is just to self document.